library/jruby

```diff
@@ -25,6 +25,10 @@ Tags: 9.4-jdk17, 9.4.6-jdk17, 9.4.6.0-jdk17
 Architectures: amd64, arm64v8
 Directory: 9.4/jdk17
 
+Tags: 9.4-jdk21, 9.4.6-jdk21, 9.4.6.0-jdk21
```

Comment:
There should be 4 new sections, right? `9.4/jdk21` `9.4/jre21` `9.3/jdk21` `9.3/jre21`